File: Number System Conversion/Decimal to Binary/Dec2Bin.cpp
#include<iostream>
using namespace std;
main()
{
	int a,b=1,i=1;
	cout<<"Enter a Number : ";
	cin>>a;
	while(a!=0)
	{
        int c = a%2;
        a /= 2;
        b += c*i;
        i *= 10;
	}
	b--;
	cout<<"Binary Value : "<<b;
}

File: Dec2Bin.java
import java.util.*;
class Dec2Bin
{
    static void main()
    {
        Scanner sc=new Scanner (System.in);
        //System.out.println("Enter a number");
        //int n=sc.nextInt();
        String s;
        int n2;
        for(int n=0;n<=16;n++)
        {
            char b[]={'0','1'};
            s="";
            n2=n;
            while(n2!=0)
            {
                int t=n2%2;
                s=b[t]+s;
                n2=n2/2;
            }
            System.out.println("result of "+n+" =  "+s);
        }
    }
}

        